gamelogic.js

```diff
@@ -1,17 +1,118 @@
+// DOEL: displayWordSoFar
+// word = "javascript" guesses = [];
+// -> "_ _ _ _ _ _ _ _ _ _ "
+// word = "javascript" guesses = ["q", "w", "e", "a", "t"];
+// -> "_ a _ a _ _ _ _ _ t "
+// STRATEGIE:
+// Een string aan elkaar plakken
+// Door over elke letter in het woord heen te loopen
+// Zit die letter er niet in: plak "_ " aan de string
+// Zit die letter er wel in: plak "<letter> " aan de string
+// - [ ] zet het eerste testje op test.only (later weer weghalen)
+// - [ ] loggen: wat is word, wat is guesses
+// - [x] javascript "splitten" word.split("") -> ['j','a','v','a','s','c','r','i','p','t']
+// - [x] Maak een output variable (met let) met de waarde lege string
+// - [x] een loop maken -> we loopen over ['j','a','v','a','s','c','r','i','p','t']
+// - [x] voo elke van letters checken: zit de letter in guesses (guess.includes(letter))?
+// - [x] if else
+// - [x] zo nee: plak "_ " aan de output variable
+// - [x] zo ja: plak "j " aan de output variable (niet j, maar de letter die we dan checken)
+// - [x] klaar? -> onze ouutput string die we hebben gemaakt returnen
+// - [x] Test geslaagd? haal test.only weg en zet het bij de volgende test erbij
+
 function displayWordSoFar(word, guesses) {
-  // WRITE ME
+  const arrayWord = word.split("");
```

Comment:
Een string kan in javascript behandeld worden als een array. De spilt is niet nodig. Je kunt 'word' meteen aanspreken.